src/pages/lobby/hooks/use-outside-click.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect, RefObject } from 'react';
+
+type RefType = RefObject<HTMLElement>;
+
+const useOutsideClick = (ref: RefType, callback: () => void) => {
+  useEffect(() => {
```

Comment:
저희 인터페이스로 하기로 했는데 아래처럼 interface로 사용할 수 있을 것 같습니당
```suggestion
  interface RefType extends RefObject<HTMLElement> {}
```